app/override/override_http.js

```diff
@@ -0,0 +1,16 @@
+export function overrideHttpUserAgent () {
+  let packageJson = require('../../package');
+  let http = require('http');
+  let originalHttpRequest = http.request;
+
+  http.request = function (options, callback) {
+    let _options = options;
+    _options.headers['User-Agent'] = `${packageJson.name}/${packageJson.version}`;
```

Comment:
Could you add `wd` and the version as well?
```
a = require('wd/package')
a.name # 'wd'
a.version # '1.13.0'
```

So, it will be like `wd/1.3.0 appium-desktop/1.20.2`

Reply:
Added the `wd` version to the User-Agent as well, on the 1.20 branch it looks like _"wd/1.13.0 appium-desktop/1.20.2-1"_